cross/irssi/PLIST

```diff
@@ -0,0 +1,6 @@
+bin:bin/irssi
+etc:etc/irssi.conf
+shr:share/doc/irssi
+shr:share/irssi/help
```

Comment:
We generally don't include helpfiles and docs to minimize package size. Could you make sure that applies here as well?
You can link to outside documentation via `HELP_URL` in spk/Makefile.